scripts/rebase-ledger.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+exit_on_error() {
+    exit_code=$1
+    if [ $exit_code -ne 0 ]; then
+        mv ./data/ledger-temp.json ./data/ledger.json
+        exit $exit_code
+    fi
+}
```

Comment:
Consider using:
`set -e` to exit the script when any command is non-zero. [manual](https://www.gnu.org/software/bash/manual/bash.html#The-Set-Builtin)
And `trap <function_name> ERR` or `trap ... EXIT` to do the cleanup. [manual](https://www.gnu.org/software/bash/manual/bash.html#index-trap)

Reply:
I was going to recommend this as well. It'll make the cleanup function so much cleaner.

Reply:
This script would just be for the sourcecred community's use, so I think the testing infra could be something as simple as a little test script that executes on some very simple ledger cases prior to actually parsing the production ledger. If those cases fail, then the script is out of date with the ledger and the production ledger modifications can't proceed.